libraries/SITL/SITL.cpp

```diff
@@ -448,7 +448,7 @@ const AP_Param::GroupInfo SIM::var_info2[] = {
     // @DisplayName: Original Position (Altitude)
     // @Description: Specifies vehicle's startup altitude (AMSL)
     // @User: Advanced
-    AP_GROUPINFO("OPOS_ALT",    53, SIM,  opos.alt, 584.0f),
+    AP_GROUPINFO("OPOS_ALT",    53, SIM,  opos.alt, 586.2938232421875),
```

Comment:
This number is slightly different. I'm not sure why the change is in this PR?

Reply:
I meant to pull this out as a separate PR.

Problem is that our vehicle is underneath the surface as far as the terrain database is concerned - we return a negative number for rangefinder...